source/Octopus.Tentacle.Client/TentacleClient.cs

```diff
@@ -50,41 +57,76 @@ public class TentacleClient : ITentacleClient
                 capabilitiesServiceV2 = tentacleServicesDecorator.Decorate(capabilitiesServiceV2);
             }
 
-            rpcCallRetryHandler = new RpcCallRetryHandler(retryDuration, TimeoutStrategy.Pessimistic);
+            var rpcCallRetryHandler = new RpcCallRetryHandler(retryDuration, TimeoutStrategy.Pessimistic);
```

Comment:
Was the intention to use the RpcCallExecutorFactory here?

Reply:
Lol, yes, thank you 😁